styles/test.scss

```
@@ -0,0 +1,10 @@
+/**
+ * Stylesheet for tests
+ */
+
+// legacy component styles
```

```suggestion
// Legacy component styles
```

Perhaps "Styles for legacy components" and "Styles for new components", to be clear that legacy/updated refers to the components and not just a different set of styles for the same components.